security/README.md

```diff
@@ -35,6 +35,7 @@ A public disclosure date is negotiated by the etcd Product Security Committee an
 ## Security Audit
 
 A third party security audit was performed by Trail of Bits, find the full report [here](SECURITY_AUDIT.pdf).
+A third fuzzing security audit was performed by Ada Logics, find the full report [here](FUZZING_AUDIT_2022.pdf).
```

Comment:
@AdamKorcz a small catch - `third party` :)

Reply:
Thanks for catching that!